python/lsst/pipe/base/graph/graph.py

```diff
@@ -54,8 +52,9 @@
 )
 
 import networkx as nx
-from lsst.daf.butler import DatasetRef, DimensionRecordsAccumulator, DimensionUniverse, Quantum
+from lsst.daf.butler import DatasetRef, DatasetType, DimensionRecordsAccumulator, DimensionUniverse, Quantum
```

Comment:
Was this import broken before somehow, or implicit, I don't see anywhere else in the diff for this file where it has been added.

Reply:
Look a few lines above. DatasetType was being explicitly imported right at the top of the file using lsst.daf.butler.core.types.

Reply:
You are right, sorry I could have sworn I didn't see that in the diff before, maybe in a different github view or something. And thanks for the fix, sometimes if I use tab complete on VSCode it will auto add the import for me, and if I don't catch it you end up with the above.